### PR TITLE
fixing picking last schedule for multiple schedules provided

### PR DESCRIPTION
**Pull request by pawelhulek**
This PR is fixing an issue with picking the last schedule from the list.

This PR allows users to pick one collection schedule based on the type of housing like 
* standalone house
* flats
* Businesses 
* etc

Also, there is a script provided to discover the supported types for a given address.

Signed-off-by: pawel.hulek <pawelhulek@gmail.com>

### CODE REVIEW

**Comment by pawelhulek** [comment]
@mampfes Can you please review? ;)

File: custom_components/waste_collection_schedule/waste_collection_schedule/service/EcoHarmonogramPL.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        schedules_response = requests.get(
+            schedules_url + "&streetId=" + street.get("id") + "&schedulePeriodId=" + sp.get("id"),
+            headers=headers)
+        schedules_response.encoding = "utf-8-sig"
+        schedules_response = schedules_response.json()
+        return schedules_response
+
+    @staticmethod
+    def fetch_streets(sp, town, street, house_number):
+        streets_response = requests.get(
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/service/EcoHarmonogramPL.py:28]: Please pass parameters in URLs using `requests` `param` argument:

https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls

Comment (pawelhulek) [reply custom_components/waste_collection_schedule/waste_collection_schedule/service/EcoHarmonogramPL.py:32]: @mampfes done - nice spot

**Comment by mampfes** [comment]
Thanks for your contribution.

**Pull request merged**